File: client/shared/models/Service.ts
export class Service {
    constructor(private _idService: number, private _name: string, private _imageURL: string, private _category: string) {
    }

    get idService(): number {
        return this._idService;
    }

    set idService(value: number) {
        this._idService = value;
    }

    get name(): string {
        return this._name;
    }

    set name(value: string) {
        this._name = value;
    }

    get imageURL(): string {
        return this._imageURL;
    }

    set imageURL(value: string) {
        this._imageURL = value;
    }

    get category(): string {
        return this._category;
    }

    set category(value: string) {
        this._category = value;
    }
}

File: server/models/service.go
package models

import "gopkg.in/guregu/null.v3"

type Service struct {
	ID       string      `db:"id" json:"uuid"`
	Name     string      `db:"name" json:"name"`
	ImageURL null.String `db:"imageURL" json:"imageURL"`
	Category string      `db:"category" json:"category"`
}
